iodata/test/data/h2o_error.wfx

```diff
@@ -56,7 +56,7 @@ Restricted HF
 1 1 1 1 1 2 2 2 3 3
 3
 </Primitive Centers>
-####<Primitive Types>
+<Primitive Types>
```

Comment:
Can you please clarify why this line was changed? I believe it was intended to test the error raised when `</Primitive Types>` end tag exists, but the starting tag does not exist...

Reply:
At that stage, the WFX module can only check if the head and tail match. For example, `FileFormatError: test.wfx:61 Expected line </Primitive Centers> but got </Primitive Centers 123>` when the tail is "</Primitive Centers 123>". But when we are using "####</Primitive Centers>", it will raise `OSError: The <Primitive Centers> section is missing!`. However, this section is there and just the wrong tail. So I think this is not a typical error and just make it correct.